packages/crypto/src/poseidon.ts

```diff
@@ -0,0 +1,87 @@
+import assert from 'assert'
```

Comment:
Where did this file come from/what are we using it for?

Reply:
it is from `circomlibjs@0.0.8` 
also don't want to depend on too many packages we don't need 😅